src/03-modality-agnostic-files.md

```diff
@@ -217,3 +217,198 @@ code organization of these scripts at the moment.
 <sup>1</sup>Storing actual source files with the data is preferred over links to
 external source repositories to maximize long term preservation (which would
 suffer if an external repository would not be available anymore).
+<!----- Conversion time: 1.344 seconds.
+
+
+Using this Markdown file:
+
+1. Cut and paste this output into your source file.
+2. See the notes and action items below regarding this conversion run.
+3. Check the rendered output (headings, lists, code blocks, tables) for proper
+   formatting and use a linkchecker before you publish this page.
+
+Conversion notes:
+
+* Docs to Markdown version 1.0β20
+* Tue Mar 24 2020 09:07:42 GMT-0700 (PDT)
+* Source doc: BIDS Extension Proposal XX (BEP0XX): Provenance
+----->
+
+## BIDS Extension Proposal XX (BEP0XX)
+## Provenance
+### version 0.0.1 (draft)
+###  Available under the CC-BY 4.0 International license
+
+Extension moderator/lead: Satra Ghosh &lt;[satra@mit.edu](mailto:satra@mit.edu)> Camille Maumet &lt;camille.maumet@inria.fr>
+
```

Comment:
lines 237 until 243 would need to be removed as well if this is a proposal to be directly integrated into the specification